data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg

```diff
@@ -2030,10 +2072,38 @@
 #endif
 #endif
 
-        [message]
-            speaker=unit
-            message= _ "Ack! What are those things!"
-        [/message]
+        [if]
```

Comment:
The failing CI check seems to be complaining about this block... but I can't see why it would be doing so.

Reply:
Wrong line number - the CI is complaining about tabs instead of spaces around line 2175, 2075.